governance/votes/Executive vote - April 1, 2022.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Template - [Executive Vote] Core Unit Budget Distributions and Vesting Streams Setup, Retroactive MKR Compensation for Long For Wisdom, Gelato Keeper Network DAI Budget Stream Setup - April 1, 2022
+summary: Core Unit Budget Distributions for GOV-001, IS-001, RWF-001; Vesting Streams Setup for GOV-001; Retroactive MKR Compensation Transfer for Long For Wisdom; Gelato Keeper Network DAI Budget Stream Setup
+date: 2022-04-01T00:00:00.000Z
+address: "$spell_address"
+---
+
+# [Executive Proposal] Core Unit Budget Distributions and Vesting Streams Setup, Retroactive MKR Compensation for Long For Wisdom, Gelato Keeper Network DAI Budget Stream Setup - April 1, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- A total of 2,433,452.30 DAI (itemized further down) will be transferred to multiple Core Units.
+- A total of 6,866,436.69 DAI (itemized further down) will be streamed to multiple Core Units.
+- The existing DAI budget stream for the Real-World Finance Core Unit (RWF-001) will be revoked.
+- Three Core Unit vesting streams for a total of 167.64 MKR (itemized further down) will be set up.
+- A total of 60 MKR will be transferred to Long For Wisdom as retroactive compensation.
+- A total of 731,000 DAI will be streamed to the Gelato Keeper Network.
```

Comment:
Just note there's a chance this might change, we are waiting for Hexonaut to get back to us

Reply:
@blimpa you need to change the figure for Gelato here

Reply:
Why are the DAI and MKR amounts in this section not in **bold** while the amounts lower down are?